sdks/python/apache_beam/coders/stream.pyx

```diff
@@ -63,7 +63,9 @@ cdef class OutputStream(object):
         break
 
   cpdef write_bigendian_int64(self, libc.stdint.int64_t signed_v):
-    cdef libc.stdint.uint64_t v = signed_v
+    self.write_bigendian_uint64(signed_v)
```

Comment:
Do you need the type casts similar to `read_bigendian_double` and `write_bigendian_double'?